packages/plugins/src/plugins/spark/constants.ts

```diff
@@ -0,0 +1,45 @@
+import {
+  EvmNetworkIdType,
+  NetworkId,
+  networks,
+} from '@sonarwatch/portfolio-core';
+import { Address } from 'viem';
+import { LendingConfig } from '../aave/types';
```

Comment:
```suggestion
import { LendingConfig as AaveLendincConfig } from '../aave/types';
```

Did you consider using an alias name on the `import` to make it obvious we're _borrowing_ this data structure from Aave ?